src/Components/Menu/MenuItem.js

```diff
@@ -6,16 +6,22 @@ import './MenuItem.css';
 export default function MenuItem(props) {
   const ctx = useContext(ItemsContext)
 
-  const [newCartItem,setNewCartItem] =useState(props.menuItem)
+  const [newCartItem,setNewCartItem] =useState(props.menuItem);
+  const [itemQty, itemQuantity] = useState(0);
 
   function qtyHandler (e) {
-    setNewCartItem((prev) =>
-      ({...prev, itemQty: +e.target.value})
-    )
+    if (e.target.value >= 0) {
```

Comment:
can we do this any other way like setting the minimum and maximum in the input field.It would be even more better if the user isnt even allowed to input any -ve values instead of checking the validations inside.

Reply:
With this change now user won't be able to put the negative values, So the minimum is set to 0, we can set someting to restrict the max.